firstfile.txt

```diff
@@ -1,2 +1,3 @@
 #just a bunch of stuff
 #next line
+#added this by editing branch 1
```

Comment:
noticed that the changes to main have not shown up in branch 1 as i was branched before then.

Reply:
i've decided to start a review. Not sure why.

Reply:
going to resolve this conversation